src/pages/AddItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const AddItem = props => {
+	return <h1>Hello from Add Item</h1>;
+};
```

Comment:
```suggestion
const AddItem = () => {
	return <h1>Hello from Add Item</h1>;
};
```

Reply:
You don't actually have to change this, but know that since you aren't actually using any props, you can just use the empty `()`.